modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPartitionSupplier.java

```diff
@@ -543,7 +543,7 @@ else if (iter != null)
      * @param row Cache data row.
      * @return Entry info.
      */
-    private GridCacheEntryInfo extractEntryInfo(CacheDataRow row) {
+    public GridCacheEntryInfo extractEntryInfo(CacheDataRow row) {
```

Comment:
I do not support code modification for tests.
You can use reflection for that purpose.